### PR TITLE
added oauth.yaml

### PR DESCRIPTION
**Pull request by cthtrifork**
Depends on:
https://github.com/trifork/cheetah-development-infrastructure/pull/5

And OS + kafka + redpanda should point at oauth simulator

### CODE REVIEW

File: docker-compose/os.yaml [hunk 1]
```diff
@@ -8,7 +10,7 @@ services:
       - discovery.type=single-node
       - bootstrap.memory_lock=true
       - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
-      - DISABLE_SECURITY_PLUGIN=true
+      #- DISABLE_SECURITY_PLUGIN=true
```
Comment (pegtrifork) [inline docker-compose/os.yaml:13]: Should this be left here? If so, make a comment in the README about it

File: docker-compose/os.yaml [hunk 2]
```diff
@@ -20,21 +22,37 @@ services:
...
       - 5602:5601
+    volumes:
+      - ../config/opensearch_dashboards/opensearch_dashboards.yml:/usr/share/opensearch-dashboards/config/opensearch_dashboards.yml
+      - ../config/opensearch_dashboards/opensearch_dashboards.crt:/usr/share/opensearch-dashboards/config/opensearch_dashboards.crt
+      - ../config/opensearch_dashboards/opensearch_dashboards.key:/usr/share/opensearch-dashboards/config/opensearch_dashboards.key
+
     environment:
       - 'OPENSEARCH_HOSTS=["http://opensearch:9200"]'
-      - DISABLE_SECURITY_DASHBOARDS_PLUGIN=true
+      #- DISABLE_SECURITY_DASHBOARDS_PLUGIN=true
```
Comment (pegtrifork) [inline docker-compose/os.yaml:55]: Should this be left here? If so, make a comment in the README about it

File: docker-compose/os.yaml [hunk 3]
```diff
@@ -1,4 +1,6 @@
 services:
+  # login fallback: admin:admin
```
Comment (pegtrifork) [inline docker-compose/os.yaml:2]: Put this in the README as well

File: config/opensearch_dashboards/opensearch_dashboards2.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+#https://opensearch.org/docs/latest/dashboards/branding/#sample-configuration
```
Comment (pegtrifork) [inline config/opensearch_dashboards/opensearch_dashboards2.yml:1]: A lot of comments here with unused code. Would be nice if this is cleaned up

**Comment by pegtrifork** [comment]
Just tried running it locally, and Kafka doesn't like the empty keys property when starting a fresh oauthsimulator.
I think we need to change the oauthsimulator to have a static key on startup that it uses for all clientids

**Comment by cthtrifork** [comment]
> Just tried running it locally, and Kafka doesn't like the empty keys property when starting a fresh oauthsimulator. I think we need to change the oauthsimulator to have a static key on startup that it uses for all clientids

Yeah I will create a jira ticket on this

**Pull request merged**